credoai/artifacts/model/classification_model.py

```diff
@@ -141,6 +152,13 @@ class DummyClassifier:
     model_like : model_like, optional
         While predictions are pre-computed, the model object, itself, may be of use for
         some evaluations (e.g. ModelProfiler).
+    binary_clf : bool, optional, default = True
```

Comment:
maybe an "auto" option would be useful? When "auto", classification will be determined by the `predict_output` vector

Reply:
See above comment. Imagine `predict_output` is [0, 0, 0...] because the event is rare and the model can't tease out when something is likely to be class 1. Auto would treat this as...binary? What if there are two rare events (class 1 and 2)? We'd miss that.